CHANGES.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+# 1.1.11 (core) / 0.17.11 (libraries)
+
+### New
+
+- [dagster-snowflake] Snowflake resource and IO manager now support private key authentication with unencrypted keys
+- [dagit] Improved query performance on Scheduled Runs page.
+- [dagit] Added or improved document (browser tab) titles throughout Dagit.
+- [dagster-dbt] The op created when using `load_assets_from_dbt*` is now configurable, allowing you to pass in specific parameters to the underlying dbt command (such as `--full-refresh`). To do so, supply config of the form `{"ops": {"run_dbt_<abcde>": {"config": {"full_refresh": True}}}}` when defining your asset job, or in Dagit.
+- A `dagster dev` which spins up both dagit and the daemon, and requires no `$DAGSTER_HOME` env var to be set
```

Comment:
Dagit changes should go above integration changes, and Dagster core changes should go above both.